tests/VObject/ITip/EvolutionTest.php

```diff
@@ -1719,6 +1717,7 @@ public function testNewEvolutionEvent()
  TRUE;LANGUAGE=en:MAILTO:dominik@fruux.com
 CREATED:20140813T153211Z
 LAST-MODIFIED:20140813T155353Z
+DTSTAMP:**ANY**
```

Comment:
@alexander-ivannikov We can check that DTSTAMP was increased and it is more than moment before parseEventForOrganizer call